docs/ref/contrib/admin/index.txt

```diff
@@ -714,6 +714,21 @@ subclass::
 
           colored_first_name.admin_order_field = '-first_name'
 
+      ``admin_order_field`` supports :ref:`field lookups <field-lookups>`.
+      For example, you can sort by values of fields on related models::
+
+          class Blog(models.Model):
+              title = models.CharField()
+              author = models.ForeignKey(Person)
+
+          class BlogAdmin(admin.ModelAdmin):
+              list_display = ('title', 'author')
+
+              def author(self, obj):
```

Comment:
I think this method clashes with the `author` model field and is ignored along with the `author.admin_order_field`.